bin/c8.js

```diff
@@ -1,35 +1,12 @@
 #!/usr/bin/env node
 'use strict'
 
-const fs = require('fs')
-const foreground = require('foreground-child')
-const { outputReport } = require('../lib/commands/report')
-const { promisify } = require('util')
-const rimraf = require('rimraf')
-const {
-  buildYargs,
-  hideInstrumenteeArgs,
-  hideInstrumenterArgs
-} = require('../lib/parse-args')
-
-const instrumenterArgs = hideInstrumenteeArgs()
-let argv = buildYargs().parse(instrumenterArgs)
-
-;(async function run () {
-  if ([
-    'check-coverage', 'report'
-  ].indexOf(argv._[0]) !== -1) {
-    argv = buildYargs(true).parse(process.argv.slice(2))
-  } else {
-    if (argv.clean) {
-      await promisify(rimraf)(argv.tempDirectory)
-      await promisify(fs.mkdir)(argv.tempDirectory, { recursive: true })
-    }
-
-    process.env.NODE_V8_COVERAGE = argv.tempDirectory
-    foreground(hideInstrumenterArgs(argv), async (done) => {
-      await outputReport(argv)
-      done()
-    })
-  }
-})()
+const semver = require('semver')
+const minNode = semver.minVersion(require('../package.json').engines.node)
+
+/* c8 ignore next 3 */
+if (semver.lt(process.version, minNode)) {
```

Comment:
`semver.lt` doesn't work in this case because it requires the second argument to be an exact version, not a semver range.

```console
$ node -p "require('semver').satisfies(process.version, '>=10.2.0')" # it works
true

$ node -p "require('semver').lt(process.version, '>=10.2.0')" # it doesn't
[eval]:1
require('semver').satisfiles('12.1.0', '>=10.2.0')
                  ^

TypeError: require(...).satisfiles is not a function
    at [eval]:1:19
    at Script.runInThisContext (vm.js:123:20)
    at Object.runInThisContext (vm.js:313:38)
    at Object.<anonymous> ([eval]-wrapper:9:26)
    at Module._compile (internal/modules/cjs/loader.js:759:30)
    at evalScript (internal/process/execution.js:79:25)
    at internal/main/eval_string.js:23:3
```

Reply:
My fault. https://github.com/bcoe/c8/pull/88/files is wrong.